integrations/pagerduty/clients/pagerduty.py

```diff
@@ -1,14 +1,18 @@
+import asyncio
 from typing import Any, AsyncGenerator
 
 import httpx
 from loguru import logger
-
-from port_ocean.utils import http_async_client
 from port_ocean.context.event import event
+from port_ocean.utils import http_async_client
+
 from .utils import get_date_range_for_last_n_months
 
 USER_KEY = "users"
 
+MAX_CONCURRENT_REQUESTS = 1
```

Comment:
why `1` ? this means no concurrent requests at all?

Reply:
The docs [says to](https://developer.pagerduty.com/docs/72d3b724589e3-rest-api-rate-limits#working-with-limits:~:text=Make%20requests%20serially%20for%20a%20given%20authentication%20token.) make requests serially as a way to avoid hitting the rate limits

Reply:
I see this
 > Account API Keys
Each Account API Key is allowed 960 requests per minute.